src/Hudl.Mjolnir/Hudl.Mjolnir.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <VersionPrefix>3.2.1</VersionPrefix>
-    <FileVersion>3.2.1</FileVersion>
+    <VersionPrefix>3.3.0</VersionPrefix>
```

Comment:
This would actually be a breaking change, since you've changed the interface. I think it's probably fine but feels like maybe too small of a change for a major bump, we should look to see if we can do it any other way.